subsys/net/lib/lwm2m/lwm2m_obj_firmware.c

```diff
@@ -395,10 +395,11 @@ static struct lwm2m_engine_obj_inst *firmware_create(uint16_t obj_inst_id)
 			     res_inst[obj_inst_id], j);
 	INIT_OBJ_RES_OPTDATA(FIRMWARE_PACKAGE_VERSION_ID, res[obj_inst_id], i,
 			     res_inst[obj_inst_id], j);
+	INIT_OBJ_RES_MULTI_OPTDATA(FIRMWARE_UPDATE_PROTO_SUPPORT_ID, res[obj_inst_id], i,
+				 res_inst[obj_inst_id], j, 1, false);
 	INIT_OBJ_RES_DATA(FIRMWARE_UPDATE_DELIV_METHOD_ID, res[obj_inst_id], i,
 			  res_inst[obj_inst_id], j, &(delivery_method[obj_inst_id]),
 			  sizeof(delivery_method[obj_inst_id]));
-
```

Comment:
I don't think there's a need to remove this empty line.

Reply:
Ah that slipped in, all fixed now.